mllib/src/test/scala/org/apache/spark/ml/feature/FValueSelectorSuite.scala

```diff
@@ -127,21 +127,18 @@ class FValueSelectorSuite extends MLTest with DefaultReadWriteTest {
   test("Test FValue selector: numTopFeatures") {
     val selector = new FValueSelector()
       .setOutputCol("filtered").setSelectorType("numTopFeatures").setNumTopFeatures(1)
-    val model = selector.fit(dataset)
```

Comment:
`model` here is never used, and a model will be fitted in following `testSelector`